monkestation/code/modules/admin/battle_royale/admin_procs.dm

```diff
@@ -0,0 +1,57 @@
+///Runs a quick set up battle royale
+/client/proc/battleRoyaleEasySetup()
+	set name = "Easy Set Up Battle Royale"
+	set category = "Admin.Fun"
+
+	if(!holder || !check_rights(R_FUN))
+		return
+
+	holder.battle_royale_easy_setup()
+
+/datum/admins/proc/battle_royale_easy_setup()
+	if(!check_rights(R_FUN))
+		return
+
+	if(!GLOB.battle_royale_controller)
+		GLOB.battle_royale_controller = new
+
+	if(GLOB.battle_royale_controller?.active)
+		to_chat(usr, span_warning("A game has already started!"))
+		return
+
+	//as LME requested saying no is as obtuse as possible(will be made less so for full merge)
+	var/input = tgui_alert(usr, "Do you want to quick start a battle royale?", "Battle royale", list("Normal(25 min max duration)", "Fast(10 min max duration)", "The lame option"))
```

Comment:
early return this, no input shouldn't count as a yes for battle royale

Reply:
no input is checked in the GAME_STATE_FINISHED check